cmd/lncli/commands.go

```diff
@@ -2117,13 +2118,26 @@ func updateChannelPolicy(ctx *cli.Context) error {
 
 	switch {
 	case ctx.IsSet("time_lock_delta"):
-		timeLockDelta = ctx.Int64("time_lock_delta")
+		timeLockDeltaUnCheck := ctx.Int64("time_lock_delta")
```

Comment:
What happens in the case that the initial value overflows an int64?

Reply:
If initial value overflows an int64, `ctx.Int64("time_lock_delta")` will return 0. I think I need use `ctx.String()` instead of `ctx.Int64()`.